challenge2-incremental/incremental_model.py

```diff
@@ -123,7 +123,7 @@ def submodel_one(global_parameters_path, df):
 
     # Load model with pretrained parameters
     model = SentimentAnalysisModel_global(input_size=modified_input_size)
-    model.load_state_dict(torch.load(global_parameters_path))
+    model.load_state_dict(state_dict)
```

Comment:
Instead of getting the weights from the path it is from the client (Which has taken it from the server)